app/Resources/views/adminscore/result.html.twig

```diff
@@ -20,14 +20,21 @@
                                 <td>
                                     <span class="text-success">{{ master[key].score|diff_format }}</span>
                                 </td>
+                                <td>
+                                    <span class="text-muted small">
+                                        {{ msg(key~"-desc", [master[key].value, master[key].value|number_format]) }}
+                                    </span>
+                                </td>
```

Comment:
With the extra column, the text is getting wrapped. Changing `stat-list` div to have `col-lg-12` instead of 6 should do it

This also doesn't show up very well on mobile, but not a big deal. I can't think of a straightforward solution off of the top of my head.